server/routes/api/landing_page.py

```diff
@@ -402,18 +411,30 @@ def has_data(data):
     return False
 
 
+def cache_prefix(*args, **kwargs):
+    return request.args.get("category") + "_" + request.args.get(
+        "cache_flag") + "_"
+
+
 @bp.route('/data/<path:dcid>')
-@cache.cached(timeout=3600 * 24, query_string=True)  # Cache for one day.
+@cache.cached(timeout=3600 * 24, query_string=True,
```

Comment:
With query_string=True, seems don't need cache_prefix? https://github.com/pallets-eco/flask-caching/blob/f11403fa0fb71e95cc73955c10b582911a613f39/flask_caching/__init__.py#L332-L339

Let me know if you see other behavior during testing

Reply:
it works with just the query string too.